swaglyrics/__main__.py

```diff
@@ -7,66 +7,104 @@
 import spotify
 from tab import app
+import chrome
 
 
 def main():
-	# 	print(r"""
-	#  ____                     _               _
-	# / ___|_      ____ _  __ _| |   _   _ _ __(_) ___ ___
-	# \___ \ \ /\ / / _` |/ _` | |  | | | | '__| |/ __/ __|
-	#  ___) \ V  V / (_| | (_| | |__| |_| | |  | | (__\__ \
-	# |____/ \_/\_/ \__,_|\__, |_____\__, |_|  |_|\___|___/
-	#                     |___/      |___/
-	# 	""")
-	print('Updating unsupported.txt from server.')
-	with open('unsupported.txt', 'w', encoding='utf-8') as f:
-		response = requests.get('http://aadibajpai.pythonanywhere.com/master_unsupported')
-		f.write(response.text)
-	print("Updated unsupported.txt successfully.")
+    # 	print(r"""
+    #  ____                     _               _
+    # / ___|_      ____ _  __ _| |   _   _ _ __(_) ___ ___
+    # \___ \ \ /\ / / _` |/ _` | |  | | | | '__| |/ __/ __|
+    #  ___) \ V  V / (_| | (_| | |__| |_| | |  | | (__\__ \
+    # |____/ \_/\_/ \__,_|\__, |_____\__, |_|  |_|\___|___/
+    #                     |___/      |___/
+    # 	""")
+    print("Updating unsupported.txt from server.")
+    with open("unsupported.txt", "w", encoding="utf-8") as f:
+        response = requests.get(
+            "http://aadibajpai.pythonanywhere.com/master_unsupported"
+        )
+        f.write(response.text)
+    print("Updated unsupported.txt successfully.")
 
-	parser = argparse.ArgumentParser(
-		description="Get lyrics for the currently playing song on Spotify. Either --tab or --cli is required.")
+    parser = argparse.ArgumentParser(
+        description="Get lyrics for the currently playing song on Spotify. Either --tab or --cli is required."
+    )
 
-	parser.add_argument('-t', '--tab', action='store_true', help='Display lyrics in a browser tab.')
-	parser.add_argument('-c', '--cli', action='store_true', help='Display lyrics in the command-line.')
+    parser.add_argument(
+        "-t", "--tab", action="store_true", help="Display lyrics in a browser tab."
+    )
+    parser.add_argument(
+        "-c", "--cli", action="store_true", help="Display lyrics in the command-line."
+    )
 
-	args = parser.parse_args()
+    args = parser.parse_args()
+    app.template_folder = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)), "templates"
+    )
+    app.static_folder = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)), "static"
+    )
+    port = 5042  # random
+    url = "http://127.0.0.1:{port}".format(port=port)
 
-	if args.tab:
-		print('Firing up a browser tab!')
-		app.template_folder = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'templates')
-		app.static_folder = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'static')
-		port = 5042  # random
-		url = "http://127.0.0.1:{port}".format(port=port)
-		threading.Timer(1.25, lambda: webbrowser.open(url)).start()
-		app.run(port=port)
+    # A Function  to initialise Chrome extension Global variables
+    def modeChecker():
```

Comment:
Hey, it will be better to use a flag instead of extra prompts so that it is more user friendly.

Reply:
Just felt taking input was more user-friendly imo.

Besides we were already passing an argument already... so multiple parameters just didn't feel good.

However, if you think it is absolutely necessary, I'll change it

Reply:
When the person is first running it, they already pass flags. So, passing just one more is more easier than waiting and answering questions. 

Also, it would be easier for a GUI as the GUI could directly run with the params instead of answering the prompts.